packages/next/client/link.js

```diff
@@ -23,14 +23,27 @@ function memoizedFormatUrl (formatUrl) {
       return lastResult
     }
 
-    const result = formatUrl(href, as)
+    const result = formatFunc(href, as)
     lastHref = href
     lastAs = as
     lastResult = result
     return result
   }
 }
 
+function formatUrl (url, isAs) {
+  url = url && typeof url === 'object'
+    ? formatWithValidation(url)
+    : url
+
+  if (process.env.__NEXT_EXPORT_TRAILING_SLASH) {
```

Comment:
Why is this located here 🤔 It's already there on line 140 and just needs to be wrapped in the if statement.

Reply:
This causes the wrong result btw, it's appending `/` on non-export pages.

Reply:
Oh gotcha 🤦‍♂️